Python/tier2_redundancy_eliminator_cases.c.h

```diff
@@ -904,6 +952,30 @@
             break;
         }
 
+        case _LOAD_ATTR_CLASS_0: {
+            _Py_UOpsSymType *attr;
+            _Py_UOpsSymType *null = NULL;
+            attr = sym_new_unknown(ctx);
+            if (attr == NULL) goto out_of_space;
+            null = sym_new_null(ctx);
+            if (null == NULL) goto out_of_space;
+            stack_pointer[-1] = attr;
+            break;
+        }
+
+        case _LOAD_ATTR_CLASS_1: {
```

Comment:
Would it make sense to change `_LOAD_ATTR_CLASS` (and all base variants of `split`) to `Py_UNREACHABLE()` in the redundancy eliminator's abstract interpreter?

Reply:
The default case already has `Py_UNREACHABLE()`.

Reply:
Done